app-arch/unar/unar-1.10.6.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs bash-completion-r1
+
+DESCRIPTION="unpacker for various archiving formats, e.g. rar v3"
+HOMEPAGE="https://unarchiver.c3.cx/"
+SRC_URI="
+	https://github.com/MacPaw/XADMaster/archive/v${PV}/XADMaster-${PV}.tar.gz
+	https://github.com/MacPaw/universal-detector/archive/1.1/universal-detector-1.1.tar.gz
+"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+RDEPEND="
+	media-sound/wavpack
+	app-arch/bzip2:=
+	dev-libs/icu:=
+	gnustep-base/gnustep-base
+	sys-libs/zlib:=
+"
+DEPEND="${RDEPEND}
+	app-arch/unzip
+	>=gnustep-base/gnustep-make-2.6.0[native-exceptions]
+	sys-devel/gcc[objc]
```

Comment:
Now this is an interesting one. I believe this should go to BDEPEND as well.

Reply:
I put it in `RDEPEND` as well because it is linked to `libobjc.so.4`. Does that make sense?

Reply:
´repoman´ didn't like this idea so I removed it from `RDEPEND`.

Reply:
I think it makes sense, then. Repoman should say "suspect" and not flag it as an error, just a warning basically asking you to "double-check to be absolutely sure".